docs/kickstart-docs.rst

```diff
@@ -1783,6 +1783,19 @@ cause a conflicting repo error.
     after reboot. Added in anaconda-22.3-1
 
 
+reqpart
+-------
+
+Automatically create partitions required by your hardware platform. These include a ``/boot/efi`` for x86_64 systems with UEFI firmware, ``biosboot`` for x86_64 systems with BIOS firmware and GPT, and ``PRePBoot`` for IBM Power Systems.
```

Comment:
I don't think these are limited to x86_64. It should also work on aarch64 UEFI systems for example.